### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by ooshoke**
- Added HTML for 'works' and 'about' sections
- Added styles for 'works' and 'about' sections
- Added relevant image files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,9 +23,114 @@ <h1 class="topic">Okolo <br> Oshoke</h1>
...
+            <ul class="project-tools">
+                <li class="tool-label">HTML/CSS</li>
+                <li class="tool-label">Ruby on Rails</li>
+                <li class="tool-label">JavaScript</li>
+            </ul>
+            <button type="button" class="view-project">See this Project &#x2192;</button>
+        </div>
+        <div class="item item4">
+            <img src="img/project_3.jpg" alt="laptop">
+            <p class="project-name">Project name goes here </p>
```
Comment (danmainah) [inline index.html]: - [x] KIndly use a header tag to your titles for the project, currently, you are using a `<p>` tag for your titles.

File: index.html [hunk 2]
```diff
@@ -23,9 +23,114 @@ <h1 class="topic">Okolo <br> Oshoke</h1>
...
+        <div class="item item2">
+            <img src="img/project_1.jpg" alt="laptop">
+            <p class="project-name">Project name goes here </p>
+            <ul class="project-tools">
+                <li class="tool-label">HTML/CSS</li>
+                <li class="tool-label">Ruby on Rails</li>
+                <li class="tool-label">JavaScript</li>
+            </ul>
+            <button type="button" class="view-project">See this Project &#x2192;</button>
+        </div>
```
Comment (danmainah) [inline index.html]: - [x] Kindly check on your design please ensure your button follow the design pattern given also I can see the `see this project ` button doesn't have a button margin like in the design, also the project technologies are wide apart check the comparison below;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

File: index.html [hunk 3]
```diff
@@ -23,9 +23,114 @@ <h1 class="topic">Okolo <br> Oshoke</h1>
...
+            <p class="entry-name">Skills</p>
+            <ul class="about-entry">
+                <li class="entry-options">Codekit</li>
+                <li class="entry-options">Codepen</li>
+                <li class="entry-options">GitHub</li>
+                <li class="entry-options">GitLab</li>
+                <li class="entry-options">Terminal</li>                
+            </ul>
+            <img src="img/skills.svg" alt="" class="back back-frame">
+        </div>
```
Comment (danmainah) [inline index.html]: Kindly ensure all sections are displayed within a reasonable window frame, I can only view all the content when the width is 360px or below kindy check on it.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi  @ooshoke , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @ooshoke  👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🌤️ 



 ## Highlights

✔️ No linter errors
✔️ Github flow was setup correctly
✔️ Professional pull request title and description

 ## Required Changes ♻️
- [x] As the previous reviewer requested, please ensure your design is fluid for width > 375 px, right now when I increase the screen width, the items stay in their position so if someone has a slightly bigger mobile screen, they will see everything attached to the left . You may reference[ this article](https://ux-alpaca.medium.com/so-what-exactly-is-the-difference-between-fixed-fluid-adaptive-and-responsive-layouts-and-why-3773272d8481) as a guide 


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ooshoke** [comment]
Good evening @zhadier I'm sorry but I'm not clear on how to do that and I haven't gotten that far in the curriculum. It's discussed in module 1, block 2 on day 3 and this doesn't appear to be a core requirement for the project I'm currently working on

**Comment by ooshoke** [comment]
> Good evening @zhadier I'm sorry but I'm not clear on how to do that and I haven't gotten that far in the curriculum. It's discussed in module 1, block 2 on day 3 and this doesn't appear to be a core requirement for the project I'm currently working on

https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md
@zhadier  Please find attached a link to the project requirements page

**Comment by zhadier** [comment]
Hey @ooshoke ,  to make your design flexible you need to use width in pecentages instead of pixels. That way when the screen size increases so will your element size. I know it is not stated explicitly you have to do this but remember that you are making the **MOBILE VERSION** of the app. And mobile is not limited to only 320 to 375 pixels. Like I mentioned, if someone has a slightly bigger screen size he will just see your portfolio on the left side of his screen. Even if you don't want to change the widths to percentages, atleast try to ensure that the content is centered on the screen for all mobile sizes. You can do that using width 100%, flex-column and align-items:center on the main container. Which is the body in your case.

**Comment by ooshoke** [comment]
> Hey @ooshoke , to make your design flexible you need to use width in pecentages instead of pixels. That way when the screen size increases so will your element size. I know it is not stated explicitly you have to do this but remember that you are making the **MOBILE VERSION** of the app. And mobile is not limited to only 320 to 375 pixels. Like I mentioned, if someone has a slightly bigger screen size he will just see your portfolio on the left side of his screen. Even if you don't want to change the widths to percentages, atleast try to ensure that the content is centered on the screen for all mobile sizes. You can do that using width 100%, flex-column and align-items:center on the main container. Which is the body in your case.

@zhadier Thanks so much for your input and help. I'll be sure to remember and implement that in other projects going forward. I've applied your recommended fix. Thanks

File: index.html [hunk 4]
```diff
@@ -23,9 +23,114 @@ <h1 class="topic">Okolo <br> Oshoke</h1>
...
+    </div>
+    <section class="gridcontainer works">
+        <h1 class="topic item1">Projects</h1>
+        <div class="item item2">
+            <img src="img/project_1.jpg" alt="laptop">
+            <h1 class="project-name">Project name goes here </h1>
+            <ul class="project-tools">
+                <li class="tool-label">HTML/CSS</li>
+                <li class="tool-label">Ruby on Rails</li>
+                <li class="tool-label">JavaScript</li>
```
Comment (koachcode) [inline index.html:38]: - Please as requested by the previous reviewer,  for all technology use, try to make it as similar to the Figma design. add some padding to it, and use the padding on the Figma.
##### Figma design 
Screenshot from 2022-06-01 02-03-14

##### Your design 

Screenshot from 2022-06-01 02-02-43

File: index.html [hunk 5]
```diff
@@ -23,9 +23,114 @@ <h1 class="topic">Okolo <br> Oshoke</h1>
...
+        </div>
+        <div class="item item3">
+            <img src="img/project_2.jpg" alt="laptop">
+            <h1 class="project-name">Project name goes here </h1>
+            <ul class="project-tools">
+                <li class="tool-label">HTML/CSS</li>
+                <li class="tool-label">Ruby on Rails</li>
+                <li class="tool-label">JavaScript</li>
+            </ul>
+            <button type="button" class="view-project">See this Project &#x2192;</button>
```
Comment (koachcode) [inline index.html:50]: - [Optional] Please implement the following button styling: Active, Hover, and Pressed, so the buttons will have these states. Please implement it on all buttons.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @ooshoke  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- No linters :+1: 
- Good PR Describtion 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**